api/source/controllers/User.js

```diff
@@ -51,6 +51,11 @@ module.exports.deleteUser = async function deleteUser (req, res, next) {
     if (elevate) {
       let userId = req.params.userId
       let projection = req.query.projection
+      let userData = await User.getUserByUserId(userId, projection, elevate, req.userObject)
```

Comment:
If the request was sent without the `statistics` projection, then the `statistics` property will not be returned. Recommend writing:

`let userData = await User.getUserByUserId(userId, ['statistics'], elevate, req.userObject)`

Reply:
ooh good catch!